src/config/config.py

```diff
@@ -0,0 +1,123 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+import os
+from pathlib import Path
+from typing import Final
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILE_NAME: Final[str] = "config.ini"
+CONFIG_DIRECTORY_NAME: Final[str] = "nina"
+
+
+def find_config_directory() -> Path:  # Pragma: no cover
+    """Get configuration directory.
+
+    Checks the operating system if it is windows or unix based. Uses enviroment
+    variables 'LOCALAPPDATA', 'XDG_CONFIG_HOME', and 'HOME' to determine
+    applications config directory. Defaults to the current working directory
+    if none of the above enviroment variables are set.
+
+    Returns
+    -------
+    Path    :
+        Path object to the configuration directory.
+    """
+    if os.name == "nt" and "LOCALAPPDATA" in os.environ:
```

Comment:
Maybe look at using `platform.system()`. https://stackoverflow.com/a/58071295.  Then we’ll have support for mac too.

Suggestion: https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations

_Seems that the original version of this vanished._

Reply:
Isnt mac config directory `~/.config`? You can also just set the enviroment variable `XDG_CONFIG_HOME` and it will always use that on unix systems.

Reply:
No.  Application Support is the «correct» path.

I already am setting that variable, but that’s only for CLI tools.

Reply:
I see, will implement something similar for Mac users then. Will probably use the `platform` package to differentiate similar to this stackoverflow. https://stackoverflow.com/questions/1854/python-what-os-am-i-running-on

Reply:
Is this the correct folder for application data? `~/Library/Application Support/nina/` @tomrtk has commented above regarding having location for the database file, would this be an appropriate place to put it?

Reply:
Well.  The database should be there and the preferences should be in `~/Library/Preferences`.

You know what, I can look more into the macOS stuff, as the only Mac user.  Just make it work for now, and we’ll finish it later.

Reply:
Sounds good, i'll open up a new issue for this.

Reply:
Nice. 👌🏻